worlds/hk/__init__.py

```diff
@@ -134,15 +134,15 @@ class DisableMapModSpoilers(Bool):
 
 
 class HKWeb(WebWorld):
-    setup_en  = Tutorial(
+    setup_en = Tutorial(
         "Mod Setup and Use Guide",
         "A guide to playing Hollow Knight with Archipelago.",
         "English",
         "setup_en.md",
         "setup/en",
         ["Ijwu"]
     )
-
+    
     setup_pt_br  = Tutorial(
```

Comment:
```suggestion
    setup_pt_br = Tutorial(
```
you also have a couple of empty lines with just spaces on them, those spaces should be removed

And there should be two empty lines between the bug report page and the HKWorld definition (so add that newline back in please)